packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -954,6 +954,76 @@ describe('cluster', () => {
     });
   });
 
+  test('addRotationMultiUser() with options', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpcWithIsolated = new ec2.Vpc(stack, 'Vpc', {
+      subnetConfiguration: [
+        { name: 'public', subnetType: ec2.SubnetType.PUBLIC },
+        { name: 'private', subnetType: ec2.SubnetType.PRIVATE_WITH_NAT },
+        { name: 'isolated', subnetType: ec2.SubnetType.PRIVATE_ISOLATED },
+      ],
+    });
+    const userSecret = new DatabaseSecret(stack, 'UserSecret', { username: 'user' });
+
+    // WHEN
+    // DB in isolated subnet (no internet connectivity)
+    const cluster = new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_MYSQL,
+      instanceProps: {
+        instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+        vpc: vpcWithIsolated,
+        vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_ISOLATED },
+      },
+    });
+
+    // Rotation in private subnet (internet via NAT)
+    cluster.addRotationMultiUser('user', {
+      secret: userSecret.attach(cluster),
+      automaticallyAfter: cdk.Duration.days(15),
+      excludeCharacters: '°_@',
+      vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_NAT },
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::SecretsManager::RotationSchedule', {
+      RotationRules: {
+        AutomaticallyAfterDays: 15,
+      },
+    });
+
+    Template.fromStack(stack).hasResourceProperties('AWS::Serverless::Application', {
+      Parameters: {
+        endpoint: {
+          'Fn::Join': ['', [
+            'https://secretsmanager.',
+            { Ref: 'AWS::Region' },
+            '.',
+            { Ref: 'AWS::URLSuffix' },
+          ]],
+        },
+        functionName: 'DatabaseuserAE6D9789',
+        vpcSubnetIds: {
+          'Fn::Join': ['', [
+            { Ref: 'VpcprivateSubnet1SubnetCEAD3716' },
+            ',',
+            { Ref: 'VpcprivateSubnet2Subnet2DE7549C' },
+          ]],
+        },
+        vpcSecurityGroupIds: {
+          'Fn::GetAtt': [
+            'DatabaseuserSecurityGroupFB2AA194',
+            'GroupId',
+          ],
+        },
+        excludeCharacters: '°_@',
+        masterSecretArn: {
+          Ref: 'DatabaseSecretAttachmentE5D1B020',
+        },
+      },
```

Comment:
I have to say - I don't understand this assertion...

1. Do we need _all_ of these properties here? Are we not worried only about the `vpcSubnetIds`?
2.
    ``` 
          'Fn::Join': ['', [
            { Ref: 'VpcprivateSubnet1SubnetCEAD3716' },
            ',',
            { Ref: 'VpcprivateSubnet2Subnet2DE7549C' },
          ]],
    ```

    Is there a clearer way we can formulate this assertion? Maybe imported subnets? It is not clear to me that these are the `PRIVATE_WITH_NAT` that we provided when creating the rotation.

Reply:
> 1. Do we need all of these properties here? Are we not worried only about the vpcSubnetIds?

We can remove `functionName` and `vpcSecurityGroupIds`. We need the other ones because we are checking that options are correctly passed (and `endpoint` is also an option).

Reply:
> 2. Is there a clearer way we can formulate this assertion?

See the propostion with `stack.resolve()`. Do we want to update the same test for single user rotation?

Reply:
Instead of weird `resolve()` tricks, can't we do this?

```ts
      // GIVEN
      vpc = ec2.Vpc.fromVpcAttributes(stack, 'Vpc', {
        vpcId: 'vpc-id',
        availabilityZones: ['az1'],
        publicSubnetIds: ['public-subnet-id-1', 'public-subnet-id-2'],
        publicSubnetNames: ['public-subnet-name-1', 'public-subnet-name-2'],
        privateSubnetIds: ['private-subnet-id-1', 'private-subnet-id-2'],
        privateSubnetNames: ['private-subnet-name-1', 'private-subnet-name-2'],
        isolatedSubnetIds: ['isolated-subnet-id-1', 'isolated-subnet-id-2'],
        isolatedSubnetNames: ['isolated-subnet-name-1', 'isolated-subnet-name-2'],
      });
    });
    const instance = new rds.DatabaseInstance(stack, 'Instance', {
      engine: rds.DatabaseInstanceEngine.MYSQL,
      vpc: vpc,
      vpcSubnets: { subnetType: ec2.SubnetType.PUBLIC },
    });

    // WHEN
    instance.addRotationMultiUser('user', {
      secret: new rds.DatabaseSecret(stack, 'DatabaseSecret', {
        username: 'user',
      }),
      vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_NAT },
    });

    // THEN
    Template.fromStack(stack).hasResourceProperties('AWS::Serverless::Application', {
      Parameters: {
        vpcSubnetIds: 'private-subnet-id-1,private-subnet-id-2',
      },
    });
```
?

Reply:
Love it!